draft-ietf-rtcweb-jsep.xml

```diff
@@ -1618,7 +1618,25 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               "max-bundle", and this is not the first m= section, or the bundle
               policy is set to "balanced", and this is not the first m= section
               for this media type, an "a=bundle-only" line.</t>
-            </list></t>
+
+	      <t>If the RtpTransceiver's RtpSender is active, an "a=rid" line
+	      for each encoding specified by the application in the RtpSenders's
+	      parameters.  If no encodings have been specified, then one RID
+	      line is generated.  The rid line is specified in <xref
+	      target="I-D.ietf-mmusic-rid"></xref>.  The direction MUST be
+	      "send".  If the application has chosen a RID value, it MUST be
```

Comment:
I added a part about about generating them.

I can't find anywhere else in the doc that says something like "The PeerConnection API MUST allow X".  Do you see one I can model?